scripts/build_scripts/build_script_all_ssl.sh

```diff
@@ -0,0 +1,318 @@
+if [ ! -d $BUILD_ROOT/bundle ]
+then
+  mkdir $BUILD_ROOT/bundle/
+fi
+if [ ! -d $BUILD_ROOT/bundle/static ]
+then
+  mkdir $BUILD_ROOT/bundle/static
+fi
+if [ ! -d $BUILD_ROOT/static/ui ]
+then
+  echo Making directory $BUILD_ROOT/static/ui
+  mkdir -p $BUILD_ROOT/static/ui
+else
+  echo Directory $BUILD_ROOT/static/ui exists
+fi
+
+# update the virtual environment
+pip install -r $BUILD_ROOT/requirements.txt
+pip install -r $BUILD_ROOT/test_requirements.txt
+
+# generate the UI
+cd $BUILD_ROOT/ui
+
+case $UN2_BUILD_OS in
+    'windows')
+        echo Compiling only UI for Windows...
+        cd $BUILD_ROOT/ui
+        yarn install
+        yarn run compile:prod
+        cp -rf $BUILD_ROOT/ui/dist/* $BUILD_ROOT/static/ui/
+        ;;
+    *)
+        echo Compiling the UI...
+        cd $BUILD_ROOT/ui
+        yarn install
+        yarn run compile:prod
+        cp -rf $BUILD_ROOT/ui/dist/* $BUILD_ROOT/static/ui/
+        ;;
+esac
+
+cp -r $BUILD_ROOT/static/assets/ $BUILD_ROOT/bundle/static/assets/
+if [ ! -d $BUILD_ROOT/bundle/static/ui ]
+then
+  mkdir -p $BUILD_ROOT/bundle/static/ui
+fi
+
+# TODO: Determine if the contents of /static/ui are a build/testing artifact.
+# If they are then consider changing the copy to a move, or explicitly remove
+# the generated files.  See commented out code below.
+cp -r $BUILD_ROOT/static/ui/. $BUILD_ROOT/bundle/static/ui
+
+# run the existing server-side API tests
+# run tests after generating the UI, because some test for presence of index.html
+WEBENV=test nosetests tests
+
+# Once the tests are complete, remove build artifacts from '/static/' directory
+# rm -rf static/ui
+
+# copy over the self-signed SSL certs
+mkdir $BUILD_ROOT/bundle/unplatform
+cp $BUILD_ROOT/unplatform/unplatform.cert.dummy.pem $BUILD_ROOT/bundle/unplatform/
+cp $BUILD_ROOT/unplatform/unplatform.key.dummy.pem $BUILD_ROOT/bundle/unplatform/
+
+# copy the Tools in modules over
+mkdir $BUILD_ROOT/bundle/modules
+cp -r $BUILD_ROOT/modules/* $BUILD_ROOT/bundle/modules/
+
+# copy the README
+cp $BUILD_ROOT/README.md $BUILD_ROOT/bundle/
+
+# copy package.json so unplatform can report its version
+cp $BUILD_ROOT/package.json bundle/
+
+# generate the latest releases of each tool, from the release branch
+if [ ! -d $BUILD_ROOT/tool-repos ]
+then
+  mkdir $BUILD_ROOT/tool-repos
+fi
+cd $BUILD_ROOT/tool-repos
+
+# Content player
+if [ ! -d content_player ]
+then
+  git clone git@github.com:CLIxIndia-Dev/content_player.git
+  cp content_player/.env.example content_player/.env
+fi
+cd content_player
+git checkout release
+git pull origin release
+yarn install
+yarn run build
+mkdir $BUILD_ROOT/bundle/static/content_player/
+cp -rf build/prod/*  $BUILD_ROOT/bundle/static/content_player/
+rm -rf $BUILD_ROOT/bundle/static/content_player/.git/
+
+# OEA Open Embedded Assessments
+cd ..
+
+if [ ! -d "OpenAssessmentsClient" ]
+then
+  git clone git@github.com:CLIxIndia-Dev/OpenAssessmentsClient.git
+fi
+cd OpenAssessmentsClient
+git checkout release
```

Comment:
This process has changed...please see:

https://github.com/CLIxIndia-Dev/unplatform_v2/blob/master/scripts/build_scripts/build_script_win32_ssl.sh#L142-L161